Project.toml

```
@@ -1,6 +1,6 @@
 name = "KernelFunctions"
 uuid = "ec8451be-7e33-11e9-00cf-bbf324bd1392"
-version = "0.10.5"
+version = "0.11.5"
```

```suggestion
version = "0.10.6"
```

This is none breaking if we've just renamed `MOInput` -> `IsotopicByFeatures`

We just need to add `Base.@deprecate_binding MOInput IsotopicByFeatures`

---

Actually, this is breaking since we changed the type parameters (and even the type of the fields). Any dispatches of the form `::MOInput{T}` and `::Vector{MOInput{T}}` won't work anymore.

---

```suggestion
version = "0.11.0"
```

---

Oops aha, lazy type 😆  thanks!